yatsm/pipeline/_pipeline.py

```diff
@@ -92,19 +92,24 @@ def __init__(self, name, func, require, output, **config):
         self.name = name
         self.func = func
         self.funcname = self.func.__name__
-        if self.func in SEGMENT_TASKS.values():
-            self.create_group = True
-        else:
-            self.create_group = False
         self.require = require
         self.output = output
         self.config = config
 
     @classmethod
-    def from_config(cls, name, config):
+    def from_config(cls, name, config, definitions=None):
+        """
+
+        .. todo::
+
+            Allow user to pass list of acceptable tasks, but default
+            to ones found by :ref:`find_tasks`
+
+        """
+        definitions = definitions or find_tasks()
```

Comment:
Previously we checked the tasks provided by the configuration file against `PIPELINE_TASKS`, which was created at module import time as a mix of "known" hard-coded tasks and tasks identified through entry points (`yatsm.pipeline.tasks`). Seems better to retrieve the tasks at initialization time so any tasks added to entry points will be available if you recreate the `Pipeline`